pulsar-broker/src/main/java/org/apache/pulsar/broker/service/AbstractReplicator.java

```diff
@@ -163,16 +175,38 @@ public synchronized void startProducer() {
 
     }
 
-    protected void checkTopicActiveAndRetryStartProducer() {
-        isLocalTopicActive().thenAccept(isTopicActive -> {
-            if (isTopicActive) {
-                startProducer();
+    protected void scheduleCheckTopicActiveAndStartProducer(final long waitTimeMs) {
```

Comment:
We'd better change the log at line:153

```
log.info("[{}] Skip the producer creation since the replicator state is : {}", replicatorId, state);
```

Reply:
Done